src/org/usfirst/frc/team2706/robot/Robot.java

```diff
@@ -130,7 +130,13 @@ public void disabledPeriodic() {
      */
     public void autonomousInit() {
         driveTrain.reset();
-      
+
+        // Activate the camera ring light
+        camera.enableRingLight(true);
+
+        // Get the bling doing autonomous patterns.
+        blingSystem.auto(); 
```

Comment:
I'm going to get the `BlingPeriodic` command doing this, no need to get the `autonomousInit` function doing it.